PWGLF/Tasks/LFNucleiBATask.cxx

```diff
@@ -62,6 +62,10 @@ struct LFNucleiBATask {
     histos.add<TH1>("tracks/h1p", "Track momentum; p (GeV/#it{c}); counts", HistType::kTH1F, {{1000, 0., 10.}});
 
     // tracks
+    histos.add<TH1>("tracks/hDCAxy", "DCAxy; #DCAxy; counts", HistType::kTH1F, {{200, -2.0, 2.0}});
```

Comment:
Here for the future you could consider adding an axis variable e.g. `    const AxisSpec dcaXyAxis{200, -2.f, 2.f, "DCA_{xy} (cm)"};`